libraries/Microsoft.Bot.Schema/OAuthCard.cs

```diff
@@ -60,6 +60,11 @@ public OAuthCard(string text = default(string), string connectionName = default(
         [JsonProperty(PropertyName = "connectionName")]
         public string ConnectionName { get; set; }
 
+        /// <summary>
+        /// The resource to try to perform token exchange with
+        /// </summary>
+        public TokenExchangeResource TokenExchangeResource { get; set; }
```

Comment:
connector api swagger (ConnectorAPI.json) should be updated to include this in the swagger no?

Reply:
ChrisM said they are abandoning the swaggers...